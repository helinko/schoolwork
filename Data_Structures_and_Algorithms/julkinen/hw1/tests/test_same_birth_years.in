R test_same_birth_years.tst
F 2
F 3
F 4
X old
X young
F 7
F 1
F 6
F 9
Q
 
