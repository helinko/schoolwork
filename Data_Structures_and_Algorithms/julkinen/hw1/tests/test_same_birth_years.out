> R test_same_birth_years.tst
> F 2
Jadzia Dax Lieutenant 2330 2335 Red
> F 3
Julian Bashir MedicalOfficer 2330 2337 Blue
> F 4
Kira Nerys Commander 2330 2339 Red
> X old
Worf Lieutenant 2330 2331 Gold
> X young
Jake Sisko Reporter 2330 2360 Red
> F 7
Quark Barkeep 2330 2346 Blue
> F 1
Worf Lieutenant 2330 2331 Gold
> F 6
Odo  Constable 2330 2342 Red
> F 9
Miles O'Brien PettyOfficer 2330 2350 Blue
> Q
